_data/communication.yml

```diff
@@ -180,17 +186,20 @@ websites:
     - name: ToutApp
       url: https://www1.toutapp.com/
       twitter: toutapp
+      facebook: toutapp
       img: toutapp.png
       tfa: No
 
     - name: WhatsApp
       url: https://www.whatsapp.com/
       twitter: whatsapp
+      facebook: WhatsApp
```

Comment:
I don't seem to be able to access the page as well, even when logged in.

![unavailable](https://cloud.githubusercontent.com/assets/3946887/19421742/f7a494bc-944a-11e6-9e43-b17f829d4844.PNG)